pkg/reconciler/v1alpha1/trigger/path/path.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package path
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	"github.com/knative/eventing/pkg/provisioners"
+)
+
+const (
+	prefix = "triggers"
+)
+
+// Generate generates the Path portion of a URI to send events to the given Trigger.
+func Generate(t *v1alpha1.Trigger) string {
+	return fmt.Sprintf("/%s/%s/%s", prefix, t.Namespace, t.Name)
+}
+
+// Parse parses the Path portion of a URI to determine which Trigger the request corresponds to. It
+// is expected to be in the form "/triggers/namespace/name".
```

Comment:
suggestion: This functionality could be moved to utils (outside of triggers). That way we can standardize the url to be /resource/namespace/name. I can easily this being used for channels once we fix the addressable and then get rid of the ExternalName service too. Standardizing will help.

Reply:
I agree that we can use this as a generic util. I/whoever writes that PR will happily move it out when the functionality is needed elsewhere. 

I don't want to move it out now because I find that doing that prematurely leads to gigantic util packages where everything is only used by one consumer anyway.